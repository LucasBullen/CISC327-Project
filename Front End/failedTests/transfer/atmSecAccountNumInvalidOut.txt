3c3
<  Enter number of account to deposit into.
---
> Enter number of account to transfer from.
5c5
< Incorrect account number format (8 digits, no leading zeros). Transfer cancelled.
---
> Account number does not exist. Transfer cancelled.
