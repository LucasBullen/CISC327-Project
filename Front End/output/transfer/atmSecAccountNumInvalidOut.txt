Select mode: atm or agent.
Logged into atm mode.
 Enter number of account to deposit into.
Enter number of account to transfer to.
Incorrect account number format (8 digits, no leading zeros). Transfer cancelled.
Logged out.
