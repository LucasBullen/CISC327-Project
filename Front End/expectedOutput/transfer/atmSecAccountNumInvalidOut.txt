Select mode: atm or agent.
Logged into atm mode.
Enter number of account to transfer from.
Enter number of account to transfer to.
Account number does not exist. Transfer cancelled.
Logged out.
